### PR TITLE
Fix/404—app styles, ember cli version upgrade

### PR DESCRIPTION
**Pull request by aklkv**
@jenweber In this PR:
- Fix for 404 error, app styles was not generated cause some one deleted default `styles/app.css`, it's important to keep it for this reasons.
- Upgraded ember cli to latest
- Other packages upgrades. (Everything sims to run smoothly).

Let me know what you think

### CODE REVIEW

**Comment by aklkv** [comment]
@jenweber I think we need travis here is well

**Comment by mansona** [comment]
@aklkv I have enabled travis for this repo 👍

**Comment by aklkv** [comment]
As a side note, I did not include hbs linting as a part of ember upgrade, after discussion with @mansona it sims to be unnecessary. But again let chat if anyone thinks differently

**Comment by mansona** [comment]
for context for anyone else: all of the templates for this app actually live in https://github.com/stonecircle/guidemaker-ember-template so it doesn't make sense to add hbs linting here.

We should think about adding it over there tho 😂

File: package.json [hunk 1]
```diff
@@ -18,40 +21,41 @@
...
     "ember-learn-search-components": "github:ember-learn/ember-learn-search-components#feature/styling",
     "ember-load-initializers": "^1.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-resolver": "^4.0.0",
-    "ember-source": "~3.3.0",
-    "ember-welcome-page": "^3.0.0",
-    "eslint-plugin-ember": "^5.0.0",
+    "ember-resolver": "^5.0.1",
+    "ember-source": "~3.4.0",
+    "ember-welcome-page": "^3.2.0",
```
Comment (sivakumar-kailasam) [inline package.json]: could you please remove `ember-welcome-page`?

**Review by sivakumar-kailasam — DISMISSED** [review]
@aklkv could you please resolve the conflict with the main branch so that we can merge this?

**Comment by aklkv** [comment]
@sivakumar-kailasam Done ✅

**Review by mansona — APPROVED** [review]
Looks good to me 🎉

**Pull request merged**